docs/blog/2021-12-01-csp.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Don't trust your Content Security Policy
```

Comment:
We should probably wordsmith this title a bit too. It'd be good to make it something that will rank well for SEO.

Generally, if we want somebody to find this while they're coding, then you'd want to use words that are closer to a query you might expect.

"Understanding the security implications of adding a Content Security Policy"

Something like that. We should talk in the chat and go through some options before we post this up. This is the #1 metric that will tell us if this post will get search traffic or not.